### ISSUE TITLE
I added a new yolov3 implementation that includes clean mAP evaluation and image augmentation and borrowed some of your functions

### ISSUE THREAD

**Issue by emadboctorx** [zzh8829/yolov3-tf2#277]
Hello, I've made a version of yolov3, [check here](https://github.com/emadboctorx/yolov3-keras-tf2) I borrowed some of your functions including the loss function, nms, transformation functions ... I added a fully vectorized mAP evaluation and image augmentation options, my implementation is very different so if you have the time, please take a look and i'll be gladly waiting for your feedback and of course feel free to modify your code to include my version of mAP if you need to.

Here's a sample of the evaluation:

<img width="856" alt="Screen Shot 2020-05-21 at 11 36 32 PM" src="https://user-images.githubusercontent.com/58062537/82609311-925dd000-9bbc-11ea-9660-9187f2b5fc5e.png">

![map](https://user-images.githubusercontent.com/58062537/82609365-b1f4f880-9bbc-11ea-9740-316ddc737a91.png)

![pr](https://user-images.githubusercontent.com/58062537/82609376-ba4d3380-9bbc-11ea-87e7-34f05d9ba24e.png)

**Comment by johnny-mueller** [comment]
@emadboctorx Is there a possibility to add your mAP additions to this repository via pull request? I think many people would benefit from this. I would be very grateful and would appreciate it.

**Comment by emadboctorx** [comment]
Adding my code here will require a lot of modifications and unfortunately i don't have the time for it, i'd love to help though, you can use my repo if you really require mAP for your work
